examples/apps/generic_client/generic_client.py

```diff
@@ -41,284 +42,334 @@
 
 def ParseCommandLine(args) :
 
-	global worker_obj
-	global worker_id
-	global workload_id
-	global in_data
-	global config
-	global mode
-	global uri
-	global address
-	global show_receipt
-	global show_decrypted_output
-	global requester_signature
-
-	parser = argparse.ArgumentParser()
-	mutually_excl_group = parser.add_mutually_exclusive_group()
-	parser.add_argument("-c", "--config",
-		help="The config file containing the Ethereum contract information",
-		type=str)
-	mutually_excl_group.add_argument("-u", "--uri",
-		help="Direct API listener endpoint, default is http://localhost:1947",
-		default="http://localhost:1947",		
-		type=str)
-	mutually_excl_group.add_argument("-a", "--address",
-		help="an address (hex string) of the smart contract (e.g. Worker registry listing)",
-		type=str)
-	parser.add_argument("-m", "--mode",
-		help="should be one of listing or registry (default)",
-		default="registry",		
-		choices={"registry", "listing"},
-		type=str)
-	parser.add_argument("-w", "--worker_id",
-		help="worker id (hex string) to use to submit a work order",
-		type=str)
-	parser.add_argument("-l", "--workload_id",
-		help='workload id (hex string) for a given worker',
-		type=str)
-	parser.add_argument("-i", "--in_data",
-		help='Input data',
-		nargs="+",
-		type=str)
-	parser.add_argument("-r", "--receipt",
-		help="If present, retrieve and display work order receipt",
-		action='store_true')
-	parser.add_argument("-o", "--decrypted_output",
-		help="If present, display decrypted output as JSON",
-		action='store_true')
-	parser.add_argument("-rs", "--requester_signature",
-		help="Enable requester signature for work order requests",
-		action="store_true")
-	options = parser.parse_args(args)
-
-	if options.config:
-		conf_files = [options.config]
-	else:
-		conf_files = [TCFHOME + \
-			"/examples/common/python/connectors/tcf_connector.toml"]
-	confpaths = [ "." ]
-	try :
-		config = pconfig.parse_configuration_files(conf_files, confpaths)
-		json.dumps(config)
-	except pconfig.ConfigurationException as e :
-		logger.error(str(e))
-		sys.exit(-1)
-
-	mode = options.mode
-
-	uri = options.uri
-	if uri:
-		config["tcf"]["json_rpc_uri"] = uri
-
-	address = options.address
-	if address:
-		if mode == "listing":
-			config["ethereum"]["direct_registry_contract_address"] = \
-				address
-		elif mode == "registry":
-			logger.error("\n Only Worker registry listing address is supported." +
+    global worker_obj
```

Comment:
This file has too many changes which is unrelated to Work order receipt implementation. Can you please move the formatting related changes to separate PR.

Reply:
Formatting changes are related to remove tab and space mix usages. I don't want to move it to separate PR.